requirements/check-avail-extras.py

```diff
@@ -0,0 +1,16 @@
+# import gcsfs
+import hydra
+import jsonargparse
+import matplotlib
+import omegaconf
+import rich
+import torchtext
+
+print(matplotlib.__version__)
+print(torchtext.__version__)
+print(omegaconf.__version__)
+print(hydra.__version__)
+print(jsonargparse.__version__)
+# todo
+# print(gcsfs.__version__)
+print(rich.__name__)
```

Comment:
> yes, but the pip list is never verified, the print here is just to have qazi testing that the import really works...

In that case, what we care about here is importing them. Not printing the version because this will be more reliably done by `pip list`.

```suggestion
```